be/src/exec/olap_scan_prepare.cpp

```diff
@@ -750,18 +750,16 @@ void OlapScanConjunctsManager::build_column_expr_predicates() {
         // otherwise we don't need this limitation.
         const SlotDescriptor* slot_desc = slots[index];
         LogicalType ltype = slot_desc->type().type;
-        if (!is_scalar_logical_type(ltype)) continue;
-        // disable on float/double type because min/max value may lose precision
-        // The fix should be on storage layer, and this is just a temporary fix.
-        if (ltype == LogicalType::TYPE_FLOAT || ltype == LogicalType::TYPE_DOUBLE) continue;
```

Comment:
why remove this?

Reply:
I add a new method to resolved this, so don't need check again